models/inorganic_volume_filter/blur/inorganic_volume_filter_blur.sql

```diff
@@ -0,0 +1,324 @@
+),
+
+wf_filter as (
+        SELECT 
+            DISTINCT 
+            t.buyer, 
+            t.seller, 
+            'wf' as filter 
+        FROM 
+        trades t 
+        LEFT JOIN 
+        {{ ref('inorganic_volume_filter_wallet_funders') }} f1 
+            ON f1.wallet = t.buyer 
+        LEFT JOIN 
+        {{ ref('inorganic_volume_filter_wallet_funders') }} f2
+            ON f2.wallet = t.seller 
+        WHERE t.project = 'blur'
+        AND f1.funder = f2.funder 
+        OR (f1.funder = t.seller OR f2.funder = t.buyer)
+),
+
+circular_buyer as (
+        SELECT 
+            *, 
+            'circular_buyer' as filter 
+        FROM 
+        (
+        SELECT 
+            COUNT(*) as cnt, 
+            token_id as nft_token_id,
+            nft_contract_address,
+            buyer 
+        FROM 
+        trades t 
+        WHERE t.project = 'blur'
+        AND token_standard = 'erc721'
+        AND buyer <> LOWER('0x39da41747a83aee658334415666f3ef92dd0d541')
+        GROUP BY 2, 3, 4 
+        ) foo 
+        WHERE cnt >= 2 
+),
+
+circular_seller as (
+        SELECT 
+            *, 
+            'circular_seller' as filter 
+        FROM 
+        (
+        SELECT 
+            COUNT(*) as cnt, 
+            token_id as nft_token_id,
+            nft_contract_address,
+            seller 
+        FROM 
+        trades t 
+        WHERE t.project = 'blur'
+        AND token_standard = 'erc721'
+        AND buyer <> LOWER('0x39da41747a83aee658334415666f3ef92dd0d541')
+        GROUP BY 2, 3, 4 
+        ) foo 
+        WHERE cnt >= 2 
+),
+
+trades_enrich as (
+        SELECT 
+            date_trunc('day', t.block_time) as day, 
+            t.block_time, 
+            t.project, 
+            t.nft_contract_address,
+            t.token_id as nft_token_id,
+            t.tx_hash, 
+            erc20.symbol as currency,
+            t.amount_raw / POW(10, erc20.decimals) as amount, 
+            p.price as usd_price, 
+            t.amount_raw / POW(10, erc20.decimals) * p.price as usd_amount, 
+            t.buyer,
+            t.seller,
+            t.unique_trade_id 
+        FROM 
+        trades t 
+        LEFT JOIN {{ source('prices', 'usd') }} p 
+            ON p.minute = date_trunc('minute', t.block_time)
+            AND p.contract_address = t.currency_contract 
+            AND p.blockchain = 'ethereum'
+            {% if not is_incremental() %}
+            AND p.minute >= '{{project_start_date}}'
+            {% endif %}
+            {% if is_incremental() %}
+            AND p.minute >= date_trunc('day', now() - interval '1 week')
+            {% endif %}
+        LEFT JOIN 
+        {{ ref('tokens_erc20') }} erc20 
+            ON t.currency_contract = erc20.contract_address
+            AND erc20.blockchain = 'ethereum'
+        WHERE t.project = 'blur'
+),
+
+filtered_trades as (
+        SELECT 
+            t.*, 
+            CASE WHEN mt.filter IS NOT NULL THEN true ELSE FALSE END as mt_filter,
```

Comment:
Minor comment: `mt.filter is not null as mt_filter` should give the same result (without the case statement)

Reply:
ah yes ty, that doesn't make any sense haha, I guess it was to avoid null values? or do I need to put the true & false statements as strings?

thank you sooo soo much for your help ser!